brickschema/graph.py

```diff
@@ -416,12 +416,12 @@ def validate(self, shape_graphs=None, default_brick_shapes=True):
             allow_warnings=True,
         )
 
-    def serve(self, address="127.0.0.1:8080"):
+    def serve(self, address="127.0.0.1:8080", ignore_prefixes = []):
         """
         Start web server offering SPARQL queries and 1-click reasoning capabilities
```

Comment:
Can you add some documentation as to what `ignore_prefixes` means and what it does?